runtime/runtime_impl_jetty12/src/main/java/com/google/apphosting/runtime/jetty/JettyServletEngineAdapter.java

```diff
@@ -118,27 +120,45 @@ public InvocationType getInvocationType() {
             return InvocationType.BLOCKING;
           }
         };
-    rpcConnector =
-        new DelegateConnector(server, "RPC") {
-          @Override
-          public void run(Runnable runnable) {
-            // Override this so that it does the initial run in the same thread.
-            // Currently, we block until completion in serviceRequest() so no point starting new
-            // thread.
-            runnable.run();
-          }
-        };
-    server.addConnector(rpcConnector);
+
+    // Don't add the RPC Connector if in HttpConnector mode.
+    if (!isHttpConnectorMode)
+    {
```

Comment:
[nit] Suggest we stick to the [K&R style](https://en.wikipedia.org/wiki/Indentation_style#K&R) since that is the convention in most other places.

Reply:
fixed